### DISCUSSION TITLE
Support for tailwind + vite

### DISCUSSION THREAD

**Discussion by adnjoo** [react-cosmos/react-cosmos#1573] [Ideas]
not sure how to enable tailwind with vite.

**Comment by predragnikolic** [comment]
Hello,

You can follow this guide
https://tailwindcss.com/docs/guides/vite


If you need to setup react cosmos to understand your tailwind config, see this:
https://github.com/react-cosmos/react-cosmos/discussions/1560_

**Reply by adnjoo** [reply]
thank you
